src/index.js

```diff
@@ -1,10 +1,10 @@
 export { default as asInput } from './asInput';
 export { default as Breadcrumb } from './Breadcrumb';
-export { default as Button } from './Button';
+export { default as Button } from './Button'; // TODO: shim from react-bootstrap
```

Comment:
Are these TODOs something that should be completed in this PR?  Or for later?

Reply:
For follow up PRs

Reply:
Adding the shims is more risky in terms of inadvertently making breaking changes so I'll keep them separate.